### ISSUE TITLE
Add more info to the network status page.

### ISSUE THREAD

**Issue by deavmi** [qaul/qaul.net_legacy#4]
More information on the network.html page would be nice.

**Comment by MathJud** [comment]
I agree! 
The latest code version shows also the IP address and has a link to each node.
What else should be there?

**Comment by deavmi** [comment]
@MathJud Mmmm.... Like an info box that can appear upon clicking on the nodes name, showing the following.
1. **Hostname**
2. **IP address**
3. **Time the node connected** to the mesh connected
4. **Current time that the node has been connected to the mesh** (I think this is called elapsed time) to the mesh for :smile:
5. **A list of the nodes the the node is neighbouring/connected too.**

But yeah, in the info box, hyperlink the hostname and IP Address to the page you said links to the node. BTW, what is this page you get when clicking on the IP address or hostname of the node, what does it contain?

<hr>

**OR**

<hr>

Even better, add all teh above mentions of the info box to a page that is hostd on the node, so when the user of a node click on the hostname or Ip address of a node, it brings up an info page of the previously clicked node.

**Comment by deavmi** [comment]
@MathJud Has the latest version been compiled and uploaded to the site yet?

**Comment by MathJud** [comment]
Nope, it is not yet in the binaries; will be in the next version though.

**Comment by deavmi** [comment]
@MathJud Okay. Thanks :smile:

**Comment by MathJud** [comment]
Okay: 
- Hostname: it shows the qaul user name, there is no DNS for that at the moment.
- IP address: implemented, will be in the next release.
- Time the node connected to the mesh connected: very difficult to say, as the mesh can contain several meshes ...
- A list of the nodes the the node is neighbouring/connected too: this list is IMHO not necessary, as it shows visually to which nodes a connection exists, and very roughly also the quality of this connection.

**Comment by deavmi** [comment]
@MathJud "_IMHO_"?

**Comment by MathJud** [comment]
IMHO = Abbr. for 'In My Humble Opinion'

**Comment by deavmi** [comment]
@MathJud Okay. Thanks anyway :smile:

**Comment by deavmi** [comment]
Maybe the current hostname viewing the network status page should be highlighted, to make kit easier for the uset to see himself in relation with the other nodes in the mesh-network. :smile:
